docs/manual.txt

```diff
@@ -956,6 +956,20 @@ In that case, the template engine used will be your theme's and the arguments yo
 as well as the global context from your ``conf.py``, are available to the template you
 are creating.
 
+.. sidebar:: The Global Context
```

Comment:
I'll do a separate branch with those. The "other metadata" should not be a sidebar anymore, either.